### PR TITLE
LPS-98417 Missing gradleTestKit

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
Hi @gabrielibson, 

Can you list the steps you did for the maven build? I dont think we have the correct scripts running.

**Comment by natocesarrego** [comment]
Hi @lawrence-lee,

I'm helping @gabrielibson to move on with this pull request and because of that I'm writing this comment. 🙂 

Can you please give us more details about your last question:

> Can you list the steps you did for the maven build?

What exactly are you looking for?

Additionally, I'd like to share with you that the **form-field** template for 7.2 isn't able to support Maven due to the limitation reported on [LPS-105072](https://issues.liferay.com/browse/LPS-105072).

Thanks in advance for your clarification.

**Comment by lawrence-lee** [comment]
Thanks @natocesarrego, 

I thought there may have been a chance where @gabrielibson  was able to get the maven build to work because of the changes in the pom.xml and the questions he had about the maven build in slack. 

I think what we can do is just not include maven support for now. We have other templates that don't support maven

**Comment by lawrence-lee** [comment]
When this gets merged to upstream, i'll rebase this https://github.com/petershin/liferay-portal/pull/968

Another change I want to mention is that we're no longer offering standalone gradle support for our templates. meaning our templates will only work inside workspace. Hopefully that should make things easier to support going forward

File: modules/sdk/project-templates/project-templates-form-field/src/test/java/com/liferay/project/templates/form/field/ProjectTemplatesFormFieldTest.java [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+			gradleProjectDir,
+			"src/main/resources/META-INF/resources/foobar.es.js",
+			"'dynamic-data-mapping-form-field-type/FieldBase/FieldBase.es';",
+			"import './foobarRegister.soy.js';",
+			"import {Config} from 'metal-state'",
+			"import templates from './foobar.soy.js';", "* Foobar Component",
+			"class Foobar extends Component", "Foobar.STATE",
+			"Soy.register(Foobar, templates);");
+
+		File mavenProjectDir = buildTemplateWithMaven(
```
Comment (gabrielibson) [inline modules/sdk/project-templates/project-templates-form-field/src/test/java/com/liferay/project/templates/form/field/ProjectTemplatesFormFieldTest.java:297]: Hi @lawrence-lee, the only thing I can speak about building the project with maven is related to this test. It is currently passing when I run it. I'm not sure if it should fail, according to the limitation mentioned by @natocesarrego. To be honest, I couldn't explain neither of both scenarios of fail or success, I just know that it's passing.

**Comment by natocesarrego** [comment]
Thanks for your update, @lawrence-lee. :) Do you still need some action from our side?

Thanks.

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
rebased here: https://github.com/lawrence-lee/liferay-portal/pull/152

**Pull request closed without merging**